lib/aullar/flair.moon

```diff
@@ -35,8 +35,27 @@ set_line_type_from_flair = (cr, flair) ->
       cr.dash = {6, 3}
 
 draw_ops = {
-  rectangle: (flair, x, y, width, height, cr) ->
+  custom: (flair, x, y, width, height, cr) ->
+    local ok, err
 
+    if flair.custom_draw
+      ok, err = pcall flair.custom_draw, flair, x, y, width, height, cr
+    else
+      ok ,err = false, "'custom_draw' function missing"
+
+    if not ok
+      -- Something went wrong in custom_draw, so draw an intensely red "missing"
+      -- cross, for visual attention and to maintain functioning of the editor.
+      log.error err
```

Comment:
We'll have to content ourselves with `io.stderr:write` here, as log is only a global on the Howl level, which is above aullar in the dependency chain.